code/Makefile

```diff
@@ -97,14 +115,27 @@ $(filter %$(TEST_E_SUFFIX), $(TEST_EXAMPLES)): %$(TEST_E_SUFFIX): %$(GEN_E_SUFFI
 	@mkdir -p $(LOG_FOLDER)
 	- $(DIFF) "stable/$*/" $(BUILD_FOLDER)$(EDIR)/ > $(LOG_FOLDER)$(EDIR)$(LOG_SUFFIX) 2>&1
 
+# actually run the tests
 test: $(GEN_EXAMPLES) $(TEST_EXAMPLES)
 	@echo ----------------------------
 	@echo Make complete, checking logs
 	@echo ----------------------------
 	@LOG_FOLDER=$(LOG_FOLDER) LOG_SUFFIX=$(LOG_SUFFIX) NOISY=$(NOISY) "$(SHELL)" $(SCRIPT_FOLDER)log_check.sh
 
+# generate pretty dot graphs too
+graphmod: check_stack
+	stack install dotgen graphmod
+
+$(filter %$(GRAPH_P_SUFFIX), $(GRAPH_PACKAGES)): %$(GRAPH_P_SUFFIX): check_stack graphmod
+	@mkdir -p $(GRAPH_FOLDER)
+	cd "drasil-$*" && find . -name '*.hs' -print | grep -v stack | xargs graphmod -q -p --no-cluster | dot -Tpdf > ../$(GRAPH_FOLDER)drasil-$*.pdf
```

Comment:
The `cd` can be folded into the `find` by replacing `.` with `"drasil-$*"` If this is done, then the `../` should be removed from the `dot` command too.

The `$*` in the `dot` command should be enclosed in `"` for confusing escaping reasons.

Reply:
changes made.